Functions.cmake

```diff
@@ -125,3 +125,27 @@ function(join _values _glue _output)
   string (REGEX REPLACE "[\\](.)" "\\1" _tmpStr "${_tmpStr}") #fixes escaping
   set (${_output} "${_tmpStr}" PARENT_SCOPE)
 endfunction(join)
+  
+
+# Wrapper function to generate source files from a thrift file with the given arguments.
+# Since Thrift 0.14, some of the files may not be generated, if they would be empty.
+# To overcome this, we manually create the source files, in case the thrift compiler does not generate them.
+function(generate_from_thrift _output_files _thrift_file _additional_args)
```

Comment:
I reverted this functionality based on your suggestion. 
This approach (removing instead of creating) indeed seems to be cleaner. 

The original idea was to "standardize"  the list of the generated files, so the developer doesn't have to worry about which file is generated or not, but such a safeguard shouldn't be necessary.

Reply:
I understand the benefit of your initial approach, but if Thrift decided to drop generating these practically empty files, we should not stick to the old behaviour.